configure.ac

```diff
@@ -1471,7 +1472,7 @@ fi
 
 dnl libevent check
 
-if test "$build_bitcoin_cli$build_bitcoind$bitcoin_enable_qt$use_tests$use_bench" != "nonononono"; then
+if test "$build_bitcoin_cli$build_bitcoind$bitcoin_enable_qt$enable_fuzz_binary$use_tests$use_bench" != "nonnoononono"; then
```

Comment:
This seems to be a typo
```suggestion
if test "$build_bitcoin_cli$build_bitcoind$bitcoin_enable_qt$enable_fuzz_binary$use_tests$use_bench" != "nononononono"; then
```

Reply:
<img src="https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.makeagif.com%2Fmedia%2F3-31-2015%2F6KMM9p.gif&f=1&nofb=1"></img>

Reply:
So the construction is not only absurd but also error-prone and hard to review, as proven here.

Reply:
Is there any way to fix this? Switching to cmake, maybe? @fanquake

Reply:
If only boolean logic was invented in this timeline !